File: src/com/kingmang/lino/parser/Generator.java
package com.kingmang.lino.parser;

public class Generator {

    public static String translateLexeme(String lexeme){
        return switch (lexeme) {
            case "Int" -> "int ";
            case "Char" -> "char ";
            case "Boolean" -> "boolean ";
            case "<>" -> "!= ";
            default -> lexeme.concat(" ");
        };
    }
}

File: parser/generator.go
package parser

func TranslateLexeme(lexeme string) string {
	switch lexeme {
	case "Int":
		return "int "
	case "Char":
		return "char "
	case "Boolean":
		return "boolean "
	case "<>":
		return "!= "
	default:
		return lexeme + " "
	}
}
